### ISSUE TITLE
Accessibility feedback for Usability Testing Prep – Authenticated Experience [Direct Deposit for EDU]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16332]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

### Must

1. **Must** recruit at least one person from each of the following demographics:
    - with a disability rating of 80% or greater (as 40% of American Veterans have an identified disability)
    - who is a woman of color (to ensure minimal representation within the sample size)
    - aged 55 or over (as age-related considerations or disabilities may present noticeable changes in behavior)
    - who does not have a college degree (to include enlisted Veterans)

<br/>

2. Rework "View payment history" — let's collaborate on this item. There was guidance provided during the [design intent collaboration](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14824#issuecomment-710358234) regarding this link. Cognitively and semantically, it should have a treatment more like the Disability & Compensation area, with the "Learn more about disability and compensation" link, rather than a dedicated row. **Update:** Since this link covers **all** payments, then it would help if the design makes that clear (I mistakenly thought it was exclusive to EDU. ).

<br/>

3. Please add a section on fraud within the research plan itself, as it is important to the study.
    1.  Consider creating a separate scenario or study that does not start on the direct deposit page itself as that _assumes_ users will know to go to direct deposits first to report being a victim of fraud.
    2.  Note: adding new tasks will require more participants or a separate study to accurately conduct analysis. Reference the Handbook of Usability Testing for more details on task order considerations.

### Consider

1. Given that it can be difficult to get desired participants and no shows, **consider** requesting at least 12 participants knowing that you might get 6.

1. **Consider** documenting sampling and facilitation risks associated with this study. For example, identifying risks associated with basing findings off a small sample size of 5 users and conducting a [sample data gap analysis post study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/how-to-apply-wizards/discovery/research/sample-data-analysis.md).*
    *Note: if the number for 5 participants was selected based off of the Nielsen Norman article [Why You Only Need to Test with 5 Users](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/), it should be documented that this "formula only holds for comparable users who will be using the site in fairly similar ways." Jacob Nielsen notes "you need to test additional users when a website has several highly distinct groups of users" which is relevant in reference to the diversity of personas in the [Voices of Veterans study](https://www.va.gov/playbook/downloads/Voices_Of_Veterans.pdf). As such, unless it can be confidently stated that all users of this experience are expected to share the same behavior (e.g. of one persona), 5 users will be insufficient to declare confidence in findings without acknowledging risk. 

1. **Consider** revisiting the question "How can you be sure of the account and routing numbers?" as it is nudging them towards a behavior that may be irrelevant to them unless it has previously been explicitly observed that users seek validation for these numbers.

1. **Consider** reframing questions in past tense whenever possible. Humans, especially those with cognitive considerations, are poor at providing accurate responses to "expected" or future events. For example, instead of "How often do you expect you'd need to update your info?", consider "When was the last time you updated your info?" or "How often have you updated your info in the past?" or if the user has not at the VA, "How often have you updated your info at your workplace in the past?"

1. **Consider** using scenario-based activities if imagination questions are presented. For example, instead of asking "What would you do if you thought this was an error?" without any pretext or context, consider providing the participant a scenario prompt to read out first. This will provide users with cognitive considerations more preparation to answer imagination-based prompts.

### Questions

1. Are there any other users beyond Veterans who should be included in this study? e.g. VSOs, dependents, etc.

1. What anticipated risks exist with the method of facilitation? e.g. talk out loud influencing navigational behavior

1. For this question in the conversation guide: "You heard there was a place on VA.gov that you could change your direct deposit info for your education payments, where would you look?", is this a leading prompt? 
    - Do Veterans encounter real life scenarios similar to this often? How would their behavior differ if they were asked to "update which bank they receive education payments" ?
    - Would it be relevant to observe where the user turns to for help? e.g. who they might contact if lost?

## Development guidance

No development guidance at this time, as I know the team already put extensive work into this section during the Profile 2.0 effort.

**Comment by 1Copenut** [comment]
This is excellent feedback! VSP is on board with all of your findings and recommendations.

**Comment by jason-gcio** [comment]
@sporeboy - Some action items per the above:
- Rework "View payment history" link...I think we need to make a ticket for this and see how that can be made more clear.
- Review and consider the items listed under "Consider" and "Questions" but I think you addressed most if not all in the pilot we just ran.
